assets/i18n/id.json

```diff
@@ -0,0 +1,262 @@
+    "dgc-stores-modal-businesses": "Berikut beberapa bisnis yang menerima Dogecoin:", 
+    "dgc-stores-modal-bitcoinshop": "BitcoinShop.us", 
+    "dgc-stores-modal-dogeoutlet": "DogeOutlet", 
+    "dgc-stores-modal-directories": "Jika belum cukup, Kamu bisa melihat direktori bisnis di bawah.", 
+    "dgc-stores-modal-dogedir": "Dogedir", 
+    "dgc-stores-modal-dogedir-desc": " - Direktori Bisnis Dogecoin.", 
+    "dgc-stores-modal-dogecoin-link": "Dogecoin.link", 
+    "dgc-stores-modal-dogecoin-link-desc": " - Direktori Dogecoin yang dikelola pengguna.", 
+    "dgc-stores-modal-disclaimer": "<strong>Catatan:</ strong> Toko-toko daring dan direktori bisnis tersebut tidak dimiliki atau dikelola oleh Dogecoin. Sementara kami buas dan mempercayainya, karena pelayanannya dapat diandalkan - harap berhati-hati dan resiko ditanggung sendiri.", 
+    "dgc-windows-wallet-modal-title": "PILIH DOMPET KAMU.", 
+    "dgc-windows-wallet-modal-multidoge": "MultiDoge", 
+    "dgc-windows-wallet-modal-multidoge-desc": "Dompet Dogecoin Yang Ringan.", 
+    "dgc-windows-wallet-modal-dogecoin-core-32bit": "Dogecoin Core. (32 bit)", 
+    "dgc-windows-wallet-modal-dogecoin-core-64bit": "Dogecoin Core. (64 bit)", 
+    "dgc-windows-wallet-modal-dogecoin-core-desc": "Dompet Dogecoin Yang \"Penuh\".", 
+    "dgc-windows-wallet-modal-dogecoin-core-guide": "Panduan Pengguna yang Dikelola Komunitas.", 
+    "dgc-windows-wallet-modal-getting-started": "Pelajari cara mengatur dompetmu dengan <a href=\"/getting-started\" target=\"_blank\">Panduan Pemula</a>.", 
+    "dgc-windows-wallet-modal-differences": "Apa perbedaan antara keduanya?", 
+    "dgc-windows-wallet-modal-sync": "Kamu harus melakukan sinkronisasi dengan blockchain sebelum menggunakan Dogecoin.", 
+    "dgc-windows-wallet-modal-ledger": "Ini berarti, Kamu memerlukan papan pencatatan Dogecoin (\"ledger\") untuk menggunakannya.", 
+    "dgc-windows-wallet-modal-light-wallet": "Multidoge adalah dompet versi \"ringan\" yang mensinkronkan dengan blockchain menggunakan proses \"pemindaian\", sehingga memungkinkan waktu penyelarasan relatif lebih singkat.",
+    "dgc-windows-wallet-modal-full-wallet": "Sementara, Dogecoin Core adalah dompet versi \"penuh\", dimana proses sinkronisasi dilakukan dengan mendownload keseluruhan data blockchain, sehingga mampu menyediakan dompet Dogecoin yang sungguh handal.", 
+    "dgc-windows-wallet-modal-prosandcons": "Keuntungan dan Kerugian?", 
+    "dgc-windows-wallet-modal-different-strokes": "Kedua dompet tersebut punya kelebihan dan kekurangan masing-masing.", 
+    "dgc-windows-wallet-modal-multidoge-pro": "MultiDoge itu dompet yang sangat ringan, jadi hanya \"memindai\" dalam blockchain sebelum menjadi tidak berguna lagi. Proses ini dilakukan sangat cepat.", 
```

Comment:
i think the translation here quite wrong compared to the original `en.json`

```
MultiDoge is a light wallet, so it only needs to \"skim\" through the blockchain before it becomes usable. This is done quickly
```

It should be
```
MultiDoge merupakan dompet yang ringan, hanya butuh untuk \"memindai\" pada blockchain sebelum dapat digunakan. Proses ini dilakukan sangat cepat.
```

Correct me if i'm wrong

Reply:
Although i still don't get the context behind "skimming through the blockchain" was. so that would be nearest translation that i can give.

If anyone can give me some context, i would appreciate that 😄

Reply:
Think of it as instead of reading everything in all the blocks, it just reads the parts it really needs. That's why it only "skims" the blockchain.

Reply:
I see, so that means it would be better if it's translated into

```
MultiDoge merupakan dompet yang ringan, hanya butuh untuk \"membaca sepintas\" pada blockchain sebelum dapat digunakan. Proses ini berlangsung sangat cepat.
```